src/aura/cmp3/cmp3.cmp

```diff
@@ -0,0 +1,8 @@
+<aura:component >
+    <aura:attribute name="contacts" type="List" />
+    <aura:handler name="init" value="{!this}" action="{!c.doInit}" />    
+    <aura:handler event="c:relateContactsEvent" action="{!c.handleApplicationEvent}"/>
+	<!-- related contacts -->
+    <aura:attribute name="test" type="String" />
```

Comment:
Never use test as a name, and got it on deploy to git, or anywhere, git keeps history tracking of each version of your code, you shod add a reasonable var name here, no matter it is not final and it is for testing propose it should be readable anyway